fastly/version.go

```diff
@@ -84,10 +84,11 @@ func (c *Client) LatestVersion(i *LatestVersionInput) (*Version, error) {
 type CreateVersionInput struct {
 	// Service is the ID of the service (required).
 	Service string
+
+	Comment string `form:"comment,omitempty"`
```

Comment:
Please comment this (irony recognized) using Go's comment string so it's rendered in the client library docs on godoc.org.

Reply:
That makes sense. I'll also add a comment to the `Comment` field of the `UpdateVersionInput` struct.